docs/trace_collection.md

```diff
@@ -290,3 +351,5 @@ The data on disk will automatically be discarded if it gets too old to ensure th
 [5]: https://docs.datadoghq.com/account_management/api-app-keys/#api-keys
 [6]: https://square.github.io/okhttp/interceptors/
 [7]: https://docs.datadoghq.com/real_user_monitoring/android/?tab=us
+[8]: https://github.com/opentracing-contrib/java-rxjava
+[9]: https://github.com/square/retrofit/tree/master/retrofit-adapters/rxjava2
```

Comment:
I think link should be pointing to `RxJava 3` version if we use it as an example.